game/magic/player/player.go

```diff
@@ -515,7 +515,8 @@ func (player *Player) LiftFog(x int, y int, radius int, plane data.Plane){
                 continue
             }
 
-            if dx * dx + dy * dy <= radius * radius {
+            // dx^2 + dy^2 <= (radius + 0.5)^2
+            if 4 * (dx * dx + dy * dy) <= 4 * radius * radius + 4 * radius + 1 {
```

Comment:
since theres a 4 on both sides of the equation isn't it redundant?
```
dx * dx + dy * dy <= radius * radius + radius + 1
```

Reply:
Yes but this avoid the need for conversion to float

Reply:
oh wait its not because of the 1 lol